pyproject.toml

```diff
@@ -0,0 +1,84 @@
+[build-system]
+requires = ["hatchling>=1.5"]
+build-backend = "hatchling.build"
+
+[project]
+name = "jupyter-sphinx"
+dynamic = ["version"]
+description = "Jupyter Sphinx Extensions"
+readme = "README.md"
+license = {file="LICENSE"}
+requires-python = ">=3.8"
+authors = [
+    { name = "Jupyter Development Team", email = "jupyter@googlegroups.com" },
+]
+dependencies = [
+    "ipykernel>=4.5.1",
+    "IPython",
+    "ipywidgets>=7.0.0",
+    "nbconvert>=5.5",
+    "nbformat",
+    "Sphinx>=7",
+]
+
+[project.urls]
+"Bug Tracker" = "https://github.com/jupyter/jupyter-sphinx/issues/"
+Documentation = "https://jupyter-sphinx.readthedocs.io"
+Homepage = "https://github.com/jupyter/jupyter-sphinx/"
+"Source Code" = "https://github.com/jupyter/jupyter-sphinx/"
```

Comment:
why setting up a "source code" page if it's the same as the "HomePage" ? Is it compulsory ?

Reply:
These links were directly pulled from `setup.py`, I'll change the HomePage to `jupyter.org`.